packages/components/src/components/checkbox/checkbox.stories.ts

```diff
@@ -32,318 +35,83 @@ export const Default = {
 };
 
 /**
- * Use the disabled attribute to disable an input checkbox. Clicks will be suppressed until the disabled state is removed
+ * Use the `size` attribute to change the size of the input checkbox. This attribute affects the font-size within the element, while the element itself remains the same size.
```

Comment:
For the preview samples please replace:
"Option" with "Checkbox".
Please do so in SIZE, ORIENTATION and LABEL sections as well

Reply:
I think this comment was not meant to be here since there's no "option" anywhere and code and figma have are in sync.